fixtures/packaging/babel-standalone.html

```diff
@@ -1,14 +0,0 @@
-<html>
```

Comment:
I forgot, we added it later. Doesn't make a difference since it tests same thing as `globals`.